lib/livebook_web/live/hub_live.ex

```diff
@@ -0,0 +1,383 @@
+
+      <div class="grow px-6 py-8 overflow-y-auto">
+        <div class="max-w-screen-md w-full mx-auto px-4 pb-8 space-y-8">
+          <div>
+            <PageHelpers.title text="Hub" socket={@socket} />
+            <p class="mt-4 text-gray-700">
+              Here you can create your Hubs.
+              Keep in mind that this configuration gets persisted and
+              will be restored on application launch.
+            </p>
+            <p class="mt-4 text-gray-700">
+              Follow the next steps to create you Hub configuration.
+            </p>
+          </div>
+
+          <div class="flex flex-col space-y-4">
+            <h2 class="text-xl text-gray-800 font-semibold pb-2 border-b border-gray-200">
+              1. Select your Hub service
+            </h2>
+
+            <div class="grid grid-cols-1 md:grid-cols-2 gap-4">
+              <.card_item id="fly" selected={@selected_hub_service} title="Fly">
+                <:logo>
+                  <.fly_logo />
+                </:logo>
+                <:headline>
+                  Connect your Livebook on Fly
+                </:headline>
+              </.card_item>
+
+              <.card_item id="enterprise" selected={@selected_hub_service} title="Livebook Enterprise">
+                <:logo>
+                  <img src="/images/logo.png" class="max-h-full max-w-[75%]" alt="Fly logo" />
+                </:logo>
+                <:headline>
+                  Write notebooks in Livebook then securely deploy and share them
+                  with your team and company with Livebook Enterprise.
+                </:headline>
+              </.card_item>
+            </div>
+          </div>
+
+          <%= if @selected_hub_service do %>
+            <div class="flex flex-col space-y-4">
+              <h2 class="text-xl text-gray-800 font-semibold pb-2 border-b border-gray-200">
+                2. Connect to your Hub with the following form
+              </h2>
+
+              <%= if @selected_hub_service == "fly" do %>
+                <.fly_form socket={@socket} data={@data} machines={@machine_options} />
+              <% end %>
+
+              <%= if @selected_hub_service == "enterprise" do %>
+                <div class="flex">
+                  <span class="text-sm font-medium">
+                    If you want to learn more, <a
+                      href="https://livebook.dev/#livebook-plans"
+                      class="pointer-events-auto text-blue-600"
+                      target="_blank"
+                    >click here</a>.
+                  </span>
+                </div>
+              <% end %>
+            </div>
+          <% end %>
+        </div>
+      </div>
+
+      <.current_user_modal current_user={@current_user} />
+    </div>
+    """
+  end
+
+  defp card_item(assigns) do
+    ~H"""
+    <div
+      id={@id}
+      class={"flex card-item flex-col " <> card_item_bg_color(@id, @selected)}
+      phx-hook="SelectHubService"
```

Comment:
I think we could use JS commands instead, something like

```
phx-click={JS.push("select_hub_service", value: @id)}
```

Reply:
TIL 🐈